Author: Albin Tonnerre <lutin@debian.org>
Description: Use /usr/bin/x-terminal-emulator instead of hardcoding xterm
Forwarded: no

--- a/src/modules/everything/evry_plug_apps.c
+++ b/src/modules/everything/evry_plug_apps.c
@@ -1390,7 +1390,7 @@
      {
         _conf = E_NEW(Module_Config, 1);
         /* setup defaults */
-        _conf->cmd_terminal = eina_stringshare_add("/usr/bin/xterm -hold -e");
+        _conf->cmd_terminal = eina_stringshare_add("/usr/bin/x-terminal-emulator -e");
         _conf->cmd_sudo = eina_stringshare_add("/usr/bin/gksudo --preserve-env");
      }
 
